packages/web_ui/src/model/instance.jsx

```diff
@@ -1,4 +1,4 @@
-import React, { useEffect, useContext, useState } from "react";
```

Comment:
Don't you still need to import React in jsx files to get babel to behave properly? Which is why the `Import React from "react";` still exists in the other files. Or does destructuring work for the same purpose?

Reply:
As far as I understand React is only referenced if there's JSX syntax like `<div>` in the file. The model files don't have any such objects in them.